packages/serverless-offline-dynamodb-streams/src/index.js

```diff
@@ -55,13 +54,17 @@ class ServerlessOfflineDynamoDBStreams {
     this.streams = [];
   }
 
+  service() {
```

Comment:
Would be best to rely on some `get` getter so there is no usage to be modified
```suggestion
  get service() {
```